pdctl/command/config_command.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 var (
-	configPrefix    = "pd/api/v1/config"
-	schedulePrefix  = "pd/api/v1/config/schedule"
-	replicatePrefix = "pd/api/v1/config/replicate"
+	configPrefix      = "pd/api/v1/config"
+	schedulePrefix    = "pd/api/v1/config/schedule"
+	replicationPrefix = "pd/api/v1/config/replicate"
+	namespacePrefix   = "pd/api/v1/config/namespace/%s"
```

Comment:
remove %s

Reply:
it is used to indicate name of namespace

Reply:
You can avoid '%s' by using string concat or _path.Join_.